src/js/view/controls/rightclick.js

```diff
@@ -89,7 +89,7 @@ export default class RightClick {
         this.el.style.top = off.y + 'px';
         this.outCount = 0;
 
-        addClass(this.playerElement, 'jw-flag-rightclick-open');
+        addClass(this.playerElement, 'jw-flag-rightclick-open jw-no-focus');
```

Comment:
shouldn't we also remove `jw-no-focus` on `hideMenu`?

Reply:
No, it will then remove the class after a couple seconds, and bring back the focus outline.
The class is automatically removed from `ui.js`, when the `playerElement` is blurred out from focus.